src/app/home/home.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-home',
+  templateUrl: './home.component.html',
+  styleUrls: ['./home.component.css']
+})
+export class HomeComponent implements OnInit {
```

Comment:
If `OnInit` life-cycle is not being used, then it can be removed

Reply:
Removed the OnInit implementation as it is not being used